### PR TITLE
Improvement to clarify composer's suggestion

### PR DESCRIPTION
**Pull request by gianarb**
## Increase composer suggest

I have a lot of problem with dependency in ZF2 :) But I'm trying to understand current choice.. :) 
In this PR I propose different message to understand the role of suggestions into the composer.json

I removed
- zendframework/zend-console  
- zendframework/zend-db

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -30,12 +30,10 @@
...
     "suggest": {
-        "ext-mongo": "*",
-        "zendframework/zend-console": "Zend\\Console component",
-        "zendframework/zend-db": "Zend\\Db component",
+        "ext-mongo": "mongodb extetension to use MongoDB writer",
         "zendframework/zend-escaper": "Zend\\Escaper component, for use in the XML formatter",
-        "zendframework/zend-mail": "Zend\\Mail component",
-        "zendframework/zend-validator": "Zend\\Validator component"
+        "zendframework/zend-mail": "Zend\\Mail component to use Email writer",
+        "zendframework/zend-validator": "Zend\\Validator component to use filter feature"
```
Comment (Martin-P) [inline composer.json]: `Zend\Validator` is for using validators, not filters, so shouldn't the message be "to use validator feature"? [`zend-filter`](https://github.com/zendframework/zend-filter) is another component.

Comment (gianarb) [inline composer.json]: Hi! Can you [see this page](https://github.com/zendframework/zend-log/search?utf8=%E2%9C%93&q=Validator)?

Zend\Validator is used to use filtering feature IMO

Comment (Martin-P) [inline composer.json]: Ok, I see. It does however do more than just filtering. [It blocks messages](https://github.com/zendframework/zend-log/blob/be6598c6e6dfa93368850b89e2d26fc4236cb29f/src/Filter/Validator.php#L50) from being logged if they do not pass validation. In that case I think the description can be a bit more explanatory like: `Zend\\Validator component to block invalid log messages`

Comment (gianarb) [inline composer.json]: Oh yes sorry.. :) I agree

File: composer.json [hunk 2]
```diff
@@ -30,12 +30,10 @@
         "phpunit/PHPUnit": "~4.0"
     },
     "suggest": {
-        "ext-mongo": "*",
-        "zendframework/zend-console": "Zend\\Console component",
-        "zendframework/zend-db": "Zend\\Db component",
```
Comment (weierophinney) [inline composer.json:35]: zend-db needs to be here; one of the possible log writers is for DB, and that uses a zend-db adapter.

File: composer.json [hunk 3]
```diff
@@ -30,12 +30,10 @@
         "phpunit/PHPUnit": "~4.0"
     },
     "suggest": {
-        "ext-mongo": "*",
-        "zendframework/zend-console": "Zend\\Console component",
```
Comment (weierophinney) [inline composer.json:34]: zend-console is used by `Zend\Log\Processor\RequestId`.

File: composer.json (hunk 2 above)
Comment (gianarb) [inline composer.json:35]: No problem I add it but in this component exist a mysql writer?

Comment (froschdesign) [inline composer.json:35]: > but in this component exist a mysql writer?

No MySQL, but a [database writer](http://framework.zend.com/manual/current/en/modules/zend.log.writers.html#writing-to-databases).

Comment (gianarb) [inline composer.json:35]: Thanks!! :)

**Comment by gianarb** [comment]
Thanks

**Pull request closed without merging**